src/storybook/stories/components/swicht.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import { action } from "@storybook/addon-actions"
+import { boolean } from "@storybook/addon-knobs"
+
+import SwitchCo from "../../../components/swichtCo"
+
+export default { title: "Swicht" }
+
+export const SwichtCo = () => (
+  <SwitchCo
+    handleChange={action("clicked")}
```

Comment:
Could you change the word from **`clicked`** to **`changed`** ? I think that sounds better